### PR TITLE
Adds Penis based fetishes

### PR DESCRIPTION
**Pull request by Rfpnj**
### Things to include in a large Pull Request:

- What is the purpose of the pull request?
Adds two new fetishes Cock addict and Dick dealer
- Give a brief description of what you changed or added.
Added fetishes, listed in phone menu, attached tease attacks, dialog was done by Nnxx
- Are any new graphical assets required?
included in PR
- Has this change been tested? If so, mention the version number that the test was based on.
yes 0.2.5.5
- So we have a better idea of who you are, what is your Discord Handle?

- If you want quick feedback, you can use @Innoxia, or jump on the Lilith's Throne discord and send Innoxia a PM.

### CODE REVIEW

File: src/com/lilithsthrone/game/character/fetishes/Fetish.java [hunk 1]
```diff
@@ -412,6 +412,67 @@ public CorruptionLevel getAssociatedCorruptionLevel() {
...
+			Colour.GENERIC_ARCANE,
+			null,
+			Util.newArrayListOfValues("<span style='color:"+ Colour.GENERIC_GOOD.toWebHexString()+ ";'>Unlocks</span> <span style='color:"+ Colour.GENERIC_SEX.toWebHexString()+ ";'>cock addict tease</span>",
+					"<span style='color:"+ Colour.GENERIC_BAD.toWebHexString()+ ";'>Weak to</span> <span style='color:"+ Colour.GENERIC_SEX.toWebHexString()+ ";'>dick dealer tease</span>"),
+			null) {
+
+		@Override
+		public String getDescription(GameCharacter owner) {
+			if (owner.isPlayer())
+				return "You are hopelessly addicted to cock. You really don't care who's providing it; all you want is for your holes to be full of delicious, meat..."
```
Comment (Pimvgd) [inline src/com/lilithsthrone/game/character/fetishes/Fetish.java]: what's the comma doing between delicious and meat?

Comment (Rfpnj) [reply src/com/lilithsthrone/game/character/fetishes/Fetish.java]: it represents some other descriptive elements not suitable for young eyes

**Comment by Pimvgd** [comment]
It would be nice, but optional, if these fetishes were used in sex positions.

**Comment by Innoxia** [comment]
I've merged this into my local code, so it will be included in 0.2.7.5. Thank you! :3

**Pull request merged**